src/makefiles/cygwin.mk

```diff
@@ -4,7 +4,8 @@ ifndef FSTROOT
 $(error FSTROOT not defined.)
 endif
 
-CXXFLAGS = -msse -msse2 -Wall -I.. -DKALDI_DOUBLEPRECISION=0  \
+DOUBLE_PRECISION = 0
```

Comment:
I propose to do it this way, as the variable has to be propagated into CUDA_FLAGS, so that the variable KALDI_DOUBLEPRECISION is set correctly during compilation of kernels (in chain/)